docs/source/howto/calculations/pw.rst

```diff
@@ -0,0 +1,259 @@
+
+.. _howto:calculations:pw:
+
+``pw.x``
+--------
+
+The ``pw.x`` code of Quantum ESPRESSO performs many different kinds of self-consistent calculations of electronic-structure properties within Density-Functional Theory (DFT),  using a plane-wave basis set and pseudopotentials.
+Examples of these properties include ground-state energy and one-electron (Kohn-Sham) orbitals, atomic forces, stresses, and structural optimization, also with variable cell.
+
+================== ===============================================================================
+Plugin class       :class:`~aiida_quantumespresso.calculations.pw.PwCalculation`
+Plugin entry point ``aiida.calculations:quantumespresso.pw``
```

Comment:
Should we add the entry point group here? It might be confusing, and it's clear that this is a calculation.

Reply:
I thought that it would be nice for completeness, but I can see how it is maybe not necessary. Will remove it then